### PR TITLE
Rework tutorial guide

### LINKED ISSUE #1: Add more detailed tutorial development guide
expand on https://github.com/uwhackweek/hackweeks-as-a-service/blob/main/docs/tutorials.md and maybe take some screenshots that document how to make a new jupyter notebook for snowex hackweek. 

a short list of tutorial development resources:
- https://geohackweek.github.io/wiki/tutorial_leads.html 
- https://carpentries.github.io/instructor-training/ 
- https://carpentries.github.io/curriculum-development/ 
- framework for high-quality software documentation (tutorial, how-to, explanation, reference) https://diataxis.fr 

another thought is to use tutorial notebook 'templates' (https://github.com/earthcube/NotebookTemplates ). EarthCube is doing this for peer-reviewed notebooks that are being published here https://www.earthcube.org/notebooks

### PR DESCRIPTION
**Pull request by scottyhq**
Closes #1.

This is a big overhaul of the tutorial development section!

### CODE REVIEW

File: docs/resources/data-management.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Data management
+
+A challenging aspect of tutorial development is how to manage datasets. Ideally, the data you use will be publicly accessible and permanent for reproducibility of your computational examples.
+
+```{important}
+Remember, a hackweek tutorial is *learning-oriented* and should guide participants through a step-wise process with a meaningful outcome.
+```
+
+## Make your tutorial data publicly accessible
+*We recommend using small (<100 MB) example datasets for your tutorial*. If you typically work with large datasets, consider working with a small subset to get your ideas across.
```
Comment (jomey) [inline docs/resources/data-management.md:10]: Could we recommend even smaller datasets?
I would be bold and recommend around 10 MB. Main concern here ist that the repository with a lot of large sample datasets for each tutorial will get big quick. This is without thinking that a tutorial can potentially have multiple versions of a sample dataset

File: docs/resources/data-management.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+You can create a repository on GitHub to publicly host the tutorial dataset, and even create a DOI for it if you want! Here is an [example for images](https://github.com/snowex-hackweek/tutorial-data), and here is an [example for n-dimensional arrays](https://github.com/scottyhq/zarrdata).
+
+
+## Stream data from a public server
+Increasingly there are ways to access data remotely in a streaming fashion so that you don't have to download files at all! At a basic level, software can read URLs to data files instead of local file paths. This is a great option to facilitate portability of your tutorial to run on different machines (hackweek JupyterHub, your personal computer, the computers of collaborators, etc.).
+
+```{warning}
+If your tutorial streams data directly from a data provider, check that scheduled server downtime for maintenance isn't planned during your presentation!
+```
```
Comment (jomey) [inline docs/resources/data-management.md:24]: Also keep in mind that URLs can change or datasets are removed permanently.

File: docs/resources/tutorial-resources.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+| Description | Links |
+| - | - |
+| SnowEx 2021 Thermal IR | (rendered)[https://snowex-hackweek.github.io/website/tutorials/thermal-ir/thermal-ir-tutorial.html], (raw)[https://github.com/snowex-hackweek/website/raw/main/book/tutorials/thermal-ir/thermal-ir-tutorial.ipynb] |
+
+
+## Delivering a tutorial
+
+Prepare your tutorials well in advance and practice it in front of a small audience for feedback. This helps ensure you have enough time to cover your full tutorial, can be an opportunity for feedback, and will give you confidence when presenting during the hackweek.
+
+### Examples
```
Comment (jomey) [inline docs/resources/tutorial-resources.md:65]: Should these examples be combined with the above and live as a dedicated section? Each subsection can say something like “See samples from past events” and link to that.

File: docs/resources/data-management.md (hunk 1 above)
Comment (scottyhq) [reply docs/resources/data-management.md:10]: I'll try to clarify that a dataset (e.g. CSV <10MB) is okay to add to a repository along with code, and >10MB should be stored elsewhere, such as a separate repository or via Zenodo

**Pull request merged**